gemd/demo/cake.py

```diff
@@ -67,10 +68,7 @@ def get_template_scope():
 
 def import_toothpick_picture() -> BytesIO:
     """Return the stream of the toothpick picture."""
-    from importlib.resources import read_binary
-    resource = read_binary("gemd.demo", "toothpick.jpg")
-
-    return BytesIO(resource)
+    return files("gemd.demo").joinpath("toothpick.jpg").open("rb")
```

Comment:
Should we rewrite this to `files("gemd.demo").joinpath("toothpick.jpg").read_bytes()` to be consistent with `strehlow_and_cook.py` and `test_parser.py`?

Reply:
It'd be nice, but that would break downstream consumers that expect a byte stream instead of bytes.